### PR TITLE
Initial Setup

### PR DESCRIPTION
**Pull request by MatthewMeacham**
This PR sets up the initial environment for the project. Notably,
* Sets up Angular v7 and initial directory structure in `/app/tda-project-ui` 
* Sets up C# Web MVC API solution targeting .NET Framework 4.7.1 and initial structure in `/api/TDAProject`
* C# Architecture includes `Core`, `Accessors`, `Engines`, and `Web` projects
* Contains an example Angular component to show how to hit back-end, get data, and display it (works off the example Api Controller provided in back-end)

## Testing
1. Follow the new `README.md`, and if it all makes sense to you and functions in the end, then this set up was successful.  

In the end, you should see this when you navigate to `http://localhost:4200/`
image

### CODE REVIEW

**Comment by CooperCorona** [comment]
I didn't have .NET Framework 4.7.1 installed. I'm pasting the download link here so we have it. I'll add it to the instructions in a later pull request. [https://dotnet.microsoft.com/download/visual-studio-sdks?utm_source=getdotnetsdk&utm_medium=referral](https://dotnet.microsoft.com/download/visual-studio-sdks?utm_source=getdotnetsdk&utm_medium=referral)

**Comment by MatthewMeacham** [comment]
@CooperCorona Good call. I've updated the README to include the .NET Framework 4.7.1 download link, the Visual Studio download link, and the Visual Studio Code download link.

**Comment by gopalbhuvana** [comment]
@MatthewMeacham @CooperCorona looks good to me. Love the example! Good to go, please clone on your local environments and make sure it works as intended. 👍

**Review by gopalbhuvana — APPROVED** [review]

File: api/TDAProject/Accessors/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
```
Comment (CooperCorona) [inline api/TDAProject/Accessors/Class1.cs:1]: Do we need all these `Class1.cs` files or can we delete them?

File: app/tda-project-ui/.gitignore [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```
Comment (CooperCorona) [inline app/tda-project-ui/.gitignore:1]: Modular .gitignores? Nice! I didn't know that was possible!

**Review by CooperCorona — APPROVED** [review]
Take a look at my comment, otherwise LGTM.

File: api/TDAProject/Accessors/Class1.cs (hunk 1 above)
Comment (MatthewMeacham) [reply api/TDAProject/Accessors/Class1.cs:1]: We could delete them--they were left in last year, so I just went with that convention

**Pull request merged**